lib/engines/hcktest/tools.rb

```diff
@@ -340,9 +334,7 @@ def get_test_info(id, key, machine, tag)
 
     def queue_test(test_id, target_key, machine, tag, support)
       retry_tools_command(__method__) do
-        act_with_tools do
-          _1.queue_test(test_id, target_key, tag, machine, tag, support)
-        end
+        act_with_tools {          _1.queue_test(test_id, target_key, tag, machine, tag, support) }
```

Comment:
Excessive indent here.

Reply:
very interesting, why rubocop ignore this))